conda_forge_tick/migrators.xsh

```diff
@@ -788,7 +788,7 @@ class Rebuild(Migrator):
         return $USERNAME + ':' + self.remote_branch()
 
     def remote_branch(self):
-        return 'rebuild'
+        return 'rebuild'+self.name.lower().replace(' ', '_')
```

Comment:
Underscores in branch names?! Gross :stuck_out_tongue_winking_eye:

Reply:
With this branch name being `rebuild_branch`, eh. PRs welcome :smile: